### PR TITLE
Unit Test fixing

### PR DESCRIPTION
**Pull request by gingters**
Made unit tests compile and green again

### CODE REVIEW

File: Thinktecture.Relay.Server.Test/Diagnostics/TraceManagerTest.cs [hunk 1]
```diff
@@ -26,12 +24,17 @@ private class OnPremiseTargetResponse : IOnPremiseTargetResponse
 		{
```
Comment (thomashilzendegen) [inline Thinktecture.Relay.Server.Test/Diagnostics/TraceManagerTest.cs]: This class seems to have no references anymore.

File: Thinktecture.Relay.Server.Test/Diagnostics/TraceManagerTest.cs [hunk 2]
```diff
@@ -109,7 +112,7 @@ public void Trace_creates_tracefiles_directory()
 				},
 				Body = new byte[] { 65, 66, 67 }
 			};
-			var onPremiseTargetResponse = new OnPremiseTargetResponse()
+			var onPremiseConnectorResponse = new OnPremiseConnectorResponse()
```
Comment (thomashilzendegen) [inline Thinktecture.Relay.Server.Test/Diagnostics/TraceManagerTest.cs]: In the previous version, a mocked class is used instead the real implementation. Shouldn't this be done again?

**Comment by thomashilzendegen** [comment]
Sometimes the following test goes red, it seems that the test code is not reliable and produces different results.

`Create_returns_Created_result_with_properly_set_DTO [0:00.046] Failed`

File: Thinktecture.Relay.Server.Test/Diagnostics/TraceManagerTest.cs (hunk 2 above)
Comment (gingters) [reply Thinktecture.Relay.Server.Test/Diagnostics/TraceManagerTest.cs]: Yes, done ;)

File: Thinktecture.Relay.Server.Test/Diagnostics/TraceManagerTest.cs (hunk 1 above)
Comment (gingters) [reply Thinktecture.Relay.Server.Test/Diagnostics/TraceManagerTest.cs]: Changed to implement the used object

**Pull request merged**